### PR TITLE
TezosRPC support

### PR DESCRIPTION
**Pull request by scodexx**
Support for Tezos RPC.

### CODE REVIEW

File: packages/rpc/src/tezos/models/TezosBlockHeader.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import BigNumber from 'bignumber.js';
+import { Transform } from 'class-transformer';
+
+export class TezosBlockHeader {
+    @Transform(value => new BigNumber(value), { toClassOnly: true })
+    level: BigNumber;
+    proto: number;
+    predecessor: string;
+    timestamp: string;
+    validation_pass: number;
```
Comment (leoneparise) [inline packages/rpc/src/tezos/models/TezosBlockHeader.ts]: We use camelcase in all properties and methods. Can you change it to camelcase and use `@Expose({ name: "validation_pass" })`?

File: packages/rpc/src/tezos/models/TezosBlockHeader.ts [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    operations_hash: string;
```
Comment (leoneparise) [inline packages/rpc/src/tezos/models/TezosBlockHeader.ts]: Use camelcase.

File: packages/rpc/src/tezos/models/TezosBlockHeader.ts [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+    fitness: number[];
+    context: string;
+    priority: number;
+    proof_of_work_nonce: string;
```
Comment (leoneparise) [inline packages/rpc/src/tezos/models/TezosBlockHeader.ts]: Use camelcase.

File: packages/rpc/src/tezos/models/TezosHead.ts [hunk 4]
```diff
@@ -0,0 +1,13 @@
+import 'reflect-metadata';
+import { Type } from 'class-transformer';
+import { TezosBlockHeader } from './TezosBlockHeader';
+
+export class TezosHead {
+    hash: string;
+
+    protocol: string;
+    chain_id: string;
```
Comment (leoneparise) [inline packages/rpc/src/tezos/models/TezosHead.ts]: Use camelcase.

File: packages/rpc/src/tezos/models/TezosBlockHeader.ts (hunk 1 above)
Comment (scodexx) [reply packages/rpc/src/tezos/models/TezosBlockHeader.ts]: Got it!.. Making the change now.

**Pull request merged**